tesseract/tesseract_planning/tesseract_motion_planners/include/tesseract_motion_planners/ompl/continuous_motion_validator.h

```diff
@@ -71,6 +71,9 @@ class ContinuousMotionValidator : public ompl::base::MotionValidator
   /** @brief The Tesseract Environment */
   tesseract_environment::Environment::ConstPtr env_;
 
+  /**< @brief The tesseract state solver */
+  tesseract_environment::StateSolver::Ptr state_solver_;
```

Comment:
Do you do anything with the `state_solver_` object besides `clone`? If not, you might consider making this a `ConstPtr` since `clone` is a `const` function

Reply:
That is probably a good idea. I will make the change.